base/special/trig.jl

```diff
@@ -725,6 +725,146 @@ function acos(x::T) where T <: Union{Float32, Float64}
     end
 end
 
+# Not used at run time, so prevent unnecessary specialization/inference.
+Base.@nospecializeinfer function _exact_string_to_floating_point((@nospecialize type::Type{<:AbstractFloat}), string::String, precision::Int = 1000)
```

Comment:
This function isn't useful. Julia's parser parses floating point numbers correctly.

Reply:
I know. The reason this function existed is just to ensure I didn't make any copy-paste error, as Sollya provides the coefficients exactly.